app/src/main/java/net/squanchy/analytics/AnalyticsModule.java

```diff
@@ -0,0 +1,17 @@
+package net.squanchy.analytics;
+
+import android.content.Context;
+
+import net.squanchy.injection.ActivityContextModule;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module(includes = {ActivityContextModule.class})
+public class AnalyticsModule {
+
+    @Provides
+    Analytics analytics(Context context) {
+        return Analytics.from(context);
```

Comment:
this should be `new Analytics(dep1, dep2, ...)` we're using dagger so that the dependencies are explicit. Other dependencies have to be provided via dagger as well :)